shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -86,7 +88,7 @@ - (void)invalidate {
 // This is left a FlutterBinaryMessenger privately for now to give people a chance to notice the
 // change. Unfortunately unless you have Werror turned on, incompatible pointers as arguments are
 // just a warning.
-@interface FlutterViewController () <FlutterBinaryMessenger>
+@interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegate>
```

Comment:
We don't document every time we implement a protocol in the codebase.  FlutterBinaryMessenger was a special case since it was a special migration step.